stripecardscan-example/build.gradle

```diff
@@ -50,6 +50,8 @@ dependencies {
     implementation "androidx.lifecycle:lifecycle-viewmodel-compose:2.4.0"
     // Integration with observables
     implementation "androidx.compose.runtime:runtime-livedata:$composeVersion"
+    implementation 'androidx.appcompat:appcompat:1.3.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
```

Comment:
nit: could you extract the version number to `build.gradle` and share it with [identity](https://github.com/stripe/stripe-android/blob/7b28155d363feb4918284f842257499660c5bc82/identity/build.gradle#L85) and [identity-example](https://github.com/stripe/stripe-android/blob/73920b9924c1b71f21aac2e7e271da5a9837753d/identity-example/build.gradle#L54) ?

Reply:
These do not need to be here in the first place. I've removed them.